src/pydocstyle/config.py

```diff
@@ -7,9 +7,9 @@
 from re import compile as re
 
 
-try:  # Python 3.x
+try:  # Python 2.x
     from ConfigParser import RawConfigParser
```

Comment:
Honestly, I would just use the backport from Python 3. The Python 2 version fo ConfigParser is *awful*.

Reply:
@Nurdok , what say you?

Reply:
Since we crossed to chasm of package dependencies, I think you should use whichever package is best for the job, even if it adds a dependency.